e2e/helper.go

```diff
@@ -23,15 +24,13 @@ func (t *DeployManager) CreateNamespace(namespace string) error {
 			Labels: label,
 		},
 	}
-	err := t.crClient.Create(context.TODO(), ns)
-	if err != nil && !errors.IsAlreadyExists(err) {
-		return err
-	}
+	err := CrClient.Create(context.TODO(), ns)
+	gomega.Expect(err).To(gomega.BeNil())
```

Comment:
earlier it was validating against two errors ( `not nil` and `IsAlreadyExists`).  But looks like the change (gomega error handling) does not cover `IsAlreadyExists`